platform_tools/odc_stac.ipynb

```diff
@@ -0,0 +1,1095 @@
+    "| [Band](https://github.com/stac-extensions/eo#band-object)         | [Measurement](https://datacube-core.readthedocs.io/en/latest/api/core-classes/measurement.html#datacube.model.Measurement)             | Pixel plane within a multi-plane asset |\n",
+    "| [Common Name](https://github.com/stac-extensions/eo#common-band-names)  | Alias                  | Refer to the same band by different  |"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2f448165-ff3a-4275-9d4e-457762e23af5",
+   "metadata": {},
+   "source": [
+    "Digital Earth Africa stores a range of data products on Amazon Web Service's Simple Cloud Storage (S3) with free public access. \n",
+    "Digital Earth Africa also provides a SpatioTemporal Asset Catalog (STAC) endpoint for listing or searching the metadata, e.g. bounding box (area of interest coordinates), collection and date and time, for this archive here: https://explorer.digitalearth.africa/stac.\n",
+    "Using the STAC endpoint provided, the `odc-stac` module gives you the ability to access Digital Earth Africa's earth observation data outside of the Analysis Sandbox, on your own resources, whether locally or on a cloud service such as Amazon Web Services (AWS), from the python environment of your choice, in the same format (as an `xarray.Dataset`) as you would in the Analysis Sandbox. You can also use `odc-stac` to load other STAC compliant earth observation data as an `xarray.Dataset`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d54379c4-7f75-46c0-ac41-672279219239",
+   "metadata": {},
+   "source": [
+    "### Getting started with odc-stac\n",
+    "\n",
+    "Instructions on how to install the `odc-stac` module into your Python environment are provided [here](https://odc-stac.readthedocs.io/en/latest/intro.html#installation). \n",
+    "\n",
+    "Example notebooks on how you can use `odc-stac` can be viewed here:\n",
+    "\n",
+    "- [Access Sentinel 2 Analysis Ready Data from Digital Earth Africa](https://odc-stac.readthedocs.io/en/latest/notebooks/stac-load-S2-deafrica.html)\n",
+    "\n",
+    "- [Access Sentinel 2 Data from AWS](https://odc-stac.readthedocs.io/en/latest/notebooks/stac-load-e84-aws.html#)\n",
+    "\n",
+    "- [Access Sentinel 2 Data on Planetary Computer](https://odc-stac.readthedocs.io/en/latest/notebooks/stac-load-S2-ms.html)\n",
+    "\n",
+    "To download and run these notebooks, visit the [odc-stac Github repository](https://github.com/opendatacube/odc-stac/tree/develop/notebooks).\n",
+    "\n",
+    "For more on the `odc-stac` see the [odc-stac documentation](https://odc-stac.readthedocs.io/en/latest) and the [odc-stac Github repository](https://github.com/opendatacube/odc-stac)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ca1555e-cc5c-450f-ac68-00295d5fb623",
+   "metadata": {},
+   "source": [
+    "## Description\n",
```

Comment:
Perhaps introduce this as "Complete example for Digital Earth Africa" rather than "Description", which will help users when looking at the docs page. Following this, maybe replace the word "notebook" with "example" as the docs page won't render as a notebook, and this could confuse the reader.